### DISCUSSION TITLE
"Todo Mode" disappears when windows are double clicked to maximize

### DISCUSSION THREAD

**Discussion by derreck503** [rxhanson/Rectangle#1097] [Ideas]
- [X] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: Ventura 13.2.1
Rectangle version: 0.66
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): Logging is blank when double clicking the window to maximize. Rectangles is being bypassed.

**Comment by Aaron-Rumpler** [comment]
Double clicking a window's title bar to 'zoom' is native macOS behaviour, and I believe it isn't something that Rectangle could override.

**Comment by rxhanson** [comment]
Thanks @Aaron-Rumpler, this is indeed the case. Closing this one out since there's nothing that can be done in Rectangle for it.

**Comment by derreck503** [comment]
[BetterSnapTool](https://folivora.ai/bettersnaptool) is able to do this: 
<img width="393" alt="Screenshot 2023-03-06 at 12 53 39 PM" src="https://user-images.githubusercontent.com/18196165/223204026-bef75f4c-3f4e-43f7-be4f-12f36061c60b.png">

**Reply by rxhanson** [reply]
Thanks for letting me know. I was thinking about it differently. I don’t have the bandwidth for this at the moment but would merge a good PR for it.
